### PR TITLE
Make fast devnet and mac-enabled versions the default

### PR DESCRIPTION
**Pull request by edmundnoble**
This will need corresponding changes in integration-tests I think.

### CODE REVIEW

**Review by frostweb3 — APPROVED** [review]

File: docker-compose.minimal.yaml [hunk 1]
```diff
@@ -71,11 +73,15 @@ services:
     - --public-key=${MINER_PUBLIC_KEY}
     - --node=bootstrap-node:1848
     - --worker=constant-delay
+    - --constant-delay-block-time=15
```
Comment (chessai) [inline docker-compose.minimal.yaml]: why do we want to increase this?

Comment (chessai) [reply docker-compose.minimal.yaml]: I guess with fast-development, it doesn't matter much. But I'd still like to know the rationale.

**Comment by chessai** [comment]
Did you test this on mac?

Also, thoughts on making the minimal images the default? And the larger images the non-default. I rarely use the non-minimal image for local testing, so that sort of "hierarchy" makes sense to me. We'd need to talk to people on slack about this.

File: docker-compose.minimal.yaml (hunk 1 above)
Comment (edmundnoble) [reply docker-compose.minimal.yaml]: Just a git issue, will revert.

**Comment by edmundnoble** [comment]
Yes it works on my Mac at home. We can also look at making the minimal images the default as well, personally I don't mind it.

**Review by chessai — APPROVED** [review]

**Pull request merged**